src/templates/index.html

```diff
@@ -1,22 +1,95 @@
+{{if .}}
+    <!-- Sidebar/menu -->
+    <nav class="w3-sidebar w3-bar-block w3-white w3-collapse w3-top" style="width:265px" id="UserSidebar">
+        <div class="w3-container w3-display-container w3-padding-16">
+            <i onclick="w3_close()" class="fa fa-remove w3-hide-large w3-button w3-display-topright"></i>
+            <h3 class="w3-wide">Life Calendar</h3>
+        </div>
+        <div class="w3-padding-64 w3-large w3-text-grey" style="font-weight:bold;">
+            <a class="w3-bar-item w3-button w3-border-bottom w3-large" href="#">
+                <img src="https://telegram.org/file/464001801/4/pPObBDJVv-M.32191.png/9963667389a3218249" style="width:26%;">
+                Egor Filatov
+            </a>
+            <a href="/friends" class="w3-bar-item w3-button">Friends</a>
+            <a href="/analyze" class="w3-bar-item w3-button">Analyze</a>
+            <a href="/settings" class="w3-bar-item w3-button">Settings</a>
+        </div>
+        <a href="/logout" class="w3-bar-item w3-button w3-padding">Logout</a>
+    </nav>
+
+    <!-- Top menu on small screens -->
+    <header class="w3-bar w3-top w3-hide-large w3-black w3-large">
+        <div class="w3-bar-item w3-padding-24 w3-wide">Life Calendar</div>
+        <a href="javascript:void(0);" class="w3-bar-item w3-button w3-padding-24 w3-right" onclick="w3_open()"><i class="fa fa-bars"></i></a>
+    </header>
+
+    <!-- Overlay effect when opening sidebar on small screens -->
+    <div class="w3-overlay w3-hide-large" onclick="w3_close()" style="cursor:pointer" title="close side menu" id="Overlay"></div>
+
+    <!-- !PAGE CONTENT! -->
+    <div class="w3-main w3-row-padding w3-margin-top" style="margin-left:265px">
+
+        <!-- Push down content on small screens -->
+        <div class="w3-hide-large" style="margin-top:83px"></div>
+
+        <!-- Day grid -->
+        <div id="dayButtons">
+            <table cols=7 class="w3-table">
+                <tr v-for="week in getDisplayedWeeks()">
+                    <td v-for="day in week">
+                        <div class="w3-col square" v-if="isDayFilled(day)"
+                             v-bind:style="{backgroundColor: getFilledDayColor(day)}"></div>
+                        <div class="w3-col square" v-else style="background-color: #a9a9a9"></div>
+                    </td>
+                </tr>
+            </table>
+        </div>
+    </div>
+
+
+    <script src="https://unpkg.com/vue"></script>
+    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+    <script src="/scripts/index.js"></script>
+    <script>
+        function w3_open() {
+            document.getElementById("UserSidebar").style.display = "block";
+            document.getElementById("Overlay").style.display = "block";
+        }
+
+        function w3_close() {
+            document.getElementById("UserSidebar").style.display = "none";
+            document.getElementById("Overlay").style.display = "none";
+        }
```

Comment:
I don't like these functions names. As far as I understand `w3` relates to the w3css framework we are using here. But these functions themselves have nothing to do with `w3`! I guess this was just copy-pasted from an example/tutorial. If so, the functions should be renamed so that their names reflect the actual purpose of these functions (open/close sidebar menu?)